### DISCUSSION TITLE
Invalid username or password supplied.

### DISCUSSION THREAD

**Discussion by amolood** [hotspotbilling/phpnuxbill#208] [Q&A]
Unable to connect to the router with the correct credentials

Router OS v : 7.15.1
Device : 
L009UiGS-2HaxD

Invalid username or password supplied.
#0 /home/user/public_html/system/autoload/Mikrotik.php(24): PEAR2\Net\RouterOS\Client->__construct()
#1 /home/user/public_html/system/controllers/routers.php(96): Mikrotik::getClient()
#2 /home/user/public_html/system/boot.php(148): include('/home/user/...')
#3 /home/user/public_html/index.php(22): require_once('/home/user/...')
#4 {main}

**Comment by Focuslinkstech** [comment]
please check your credentials very well and make sure no spaces in-between your characters, and try again
if you think its a bug kindly report it here. [Bug Report](https://t.me/phpnuxbill/6)
